frontend/src/lib/components/CommandBar/SearchBarTab.tsx

```diff
@@ -19,13 +19,14 @@ export const SearchBarTab = ({ tab, inputRef }: SearchBarTabProps): JSX.Element
 
     return (
         <div
-            className={`SearchBarTab flex items-center px-4 py-2 cursor-pointer text-xs whitespace-nowrap border-t-2 ${
+            className={`SearchBarTab flex items-center px-4 py-2 cursor-pointer text-xs whitespace-nowrap border-r-2 ${
                 isActive ? 'SearchBarTab__active font-bold border-primary-3000' : 'border-transparent'
             }`}
             onClick={() => {
                 setActiveTab(tab)
                 inputRef.current?.focus()
             }}
+            style={tab === Tab.All ? { height: 37 } : {}}
```

Comment:
Does this have to be 37 pixels exactly? `w-9` would be 36px and could be added as a class and not need the style lint exception

Reply:
<img width="269" alt="Screenshot 2023-12-04 at 11 43 25" src="https://github.com/PostHog/posthog/assets/1851359/7297091e-7c11-4c46-99de-74f4fd9dd747">

37px looks just a tad bit better here, since this is used for making the border the same height as the input. Won't be necessary if the indicator is on the left.